tools_galaxyp.yaml

```diff
@@ -156,3 +159,32 @@ tools:
   - name: proteomics_moff
     owner: galaxyp
     tool_panel_section_label: Proteomics
+
+  - name: regex_find_replace
```

Comment:
@jraysajulga do you know offhand why this tool is stricly needed and can not be replaces by https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_line/1.1.1 ... this is my last comment promised :)

Reply:
@bgruening concatenate_multiple_datasets can be replaced by another tool so this can be ignored.

As for regex_find_replace, the main issue is that **Replace Text in Entire Line** is unable to take in multiple regex's and replacements. 

In Regex Find and Replace, it is easy to do multiple replacements. We use at least three regular expressions for our workflow. 

Example:
![image](https://user-images.githubusercontent.com/32586466/48634613-95ef1600-e98b-11e8-9f43-5dd4f8955fe0.png)

Reply:
@jraysajulga can you not simply run 3 tools of the classic regex-replace? Or do I miss something?

Reply:
@bgruening We could do that if needed.

Is that okay,  @subinamehta?

Reply:
We could that, but wouldn't the tool be more useful if people have multiple conditions to be fulfilled?

Reply:
@subinamehta yes. But we should rather update the old tool and this is not a critical missing feature or is it. I don't have hard feelings about this. Sorry, I just try to avoid many tools with the same functionality in Galaxy. I would rather try to maintain as less as possible tools over the years to come :)

Reply:
@subinamehta We can raise that as a requested functionality in the **Replace Text in entire line** tool. Would that be a GitHub issue?

@bgruening Understandable. I have removed the regex_find_replace tool from yams file.